### ISSUE TITLE
⌨ Accessibility Issue -  how does a screen reader user know when "edit a pool advertisement" form is complete

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5493]
## ❌ Current behaviour (issue)
Screen reader users cannot tell what to expect in this form and when they have reached the end of the "edit a pool advertisement" form fields. 
### AT configuration
 JAWS, Windows 10, Edge

### How to reproduce
Open JAWS 
Log in as an admin
Turn away from your screen using only your auditory senses 
Confirm how many form components you will need to fill out to ensure that the form is fully complete 

## ✔ Expected behaviour (fix)
 Screen reader is alerted to the number of sections contained in this UI in the page introduction text. 
![image.png](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/7071c3ea-2b7f-462e-a48c-9a352e6868f3)

### Acceptance Criteria
- [ ] Introductory text tells the reader how many sections there are in this form to complete.

**Comment by esizer** [comment]
Can we add a toast letting users know when all criteria for publishing have been met?

**Comment by mnigh** [comment]
> Can we add a toast letting users know when all criteria for publishing have been met?

@RM-1978 and @Jerryescandon, is this an acceptable solution?

**Comment by RM-1978** [comment]
@esizer I think we can if we can confirm that we are following[ WCAG 2 level AA accessibility guidelines](https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be)

**Comment by gobyrne** [comment]
@Jerryescandon will your Pool Edit/Creation page update address this?

**Comment by Jerryescandon** [comment]
I think it does, this is the current design for it, which includes a description of the page, at the bottom we include a "go back to previous page" too.

<details><summary>Screenshot</summary>
<p>

![Frame 40](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/3133353a-c77f-4425-bcaf-ceb977828f08)

</p>
</details>

**Comment by gobyrne** [comment]
Ok, let's close this issue than. It should be fixed with your update.
